diff --git a/node_modules/@uniswap/v3-sdk/dist/constants.d.ts b/node_modules/@uniswap/v3-sdk/dist/constants.d.ts
index fb8323c..984a669 100644
--- a/node_modules/@uniswap/v3-sdk/dist/constants.d.ts
+++ b/node_modules/@uniswap/v3-sdk/dist/constants.d.ts
@@ -1,6 +1,6 @@
-export declare const FACTORY_ADDRESS = "0x1F98431c8aD98523631AE4a59f267346ea31F984";
+export declare const FACTORY_ADDRESS = "0x8C6f64642bc9433D57C505A448248f445F7DDee7";
 export declare const ADDRESS_ZERO = "0x0000000000000000000000000000000000000000";
-export declare const POOL_INIT_CODE_HASH = "0xe34f199b19b2b4f47f68442619d555527d244f78a3297ea89325f843f87b8b54";
+export declare const POOL_INIT_CODE_HASH = "0xd5658b6bf2513270a9a59c7788cd0ddd387af482fcc6687f0c246d0e7e35747d";
 /**
  * The default factory enabled fee amounts, denominated in hundredths of bips.
  */
